pkg/apis/observability/v1alpha1/types.go

```diff
@@ -94,5 +120,5 @@ type OperatorsSpec struct {
 	// When the capability is enabled, the install is set to true, otherwise it is set to false.
 	// +optional
 	// +kubebuilder:validation:Optional
-	Install bool `json:"install,omitempty"`
+	Install *bool `json:"install,omitempty"`
```

Comment:
Not keen on names "enabled" and "operators" but agree to disagree.
It does not make sense for "operators" to be an extension point, but "enabled" is not. 
We talked about adding "pattern" as an extension point for the enabled deployment, the extension point for the operator installation shouldn't be a separate thing. E.g. 
```
  Install bool  // If true, install the operators required for this capabillity
  Enable bool // If true, deploy the resources to enable this capability
  Pattern string // Use the named pattern to define a specific configuration for operators and resources 
                            // Optional: if not specified a default pattern will be used.
```